package.json

```diff
@@ -55,16 +61,16 @@
     "vite-aliases": "^0.8.6"
   },
   "peerDependencies": {
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
-  },
-  "dependencies": {
     "@emotion/react": "^11.7.1",
     "@emotion/styled": "^11.6.0",
     "@mui/icons-material": "^5.2.4",
     "@mui/material": "^5.2.4",
     "@mui/styles": "^5.2.3",
     "@mui/system": "^5.2.6",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
```

Comment:
@cooper667 correct me if I am wrong but I think that we have react and react-dom as peer here? Or are you suggesting that the order that they appear is very important?

Reply:
yea, you're correct i misread the diff 😊 

![image](https://user-images.githubusercontent.com/3502798/148980437-f8114fe8-03c6-4d0a-9ce6-3991bb18ba87.png)